components/nimbus/src/nimbus.udl

```diff
@@ -239,15 +239,25 @@ interface NimbusClient {
     [Throws=NimbusError]
     sequence<AvailableExperiment> get_available_experiments();
 
-    /// Getter and setter for user's participation in all experiments.
+    /// Getter and setter for user's participation in experiments only.
     /// Possible values are:
-    /// * `true`: the user will not enroll in new experiments, and opt out of all existing ones.
-    /// * `false`: experiments proceed as usual.
+    /// * `true`: the user will enroll in experiments as usual.
+    /// * `false`: the user will not enroll in new experiments, and opt out of all existing ones.
     [Throws=NimbusError]
-    boolean get_global_user_participation();
+    boolean get_experiment_participation();
 
     [Throws=NimbusError]
-    sequence<EnrollmentChangeEvent> set_global_user_participation(boolean opt_in);
+    sequence<EnrollmentChangeEvent> set_experiment_participation(boolean opt_in);
+
+    /// Getter and setter for user's participation in rollouts.
+    /// Possible values are:
+    /// * `true`: the user will enroll in rollouts as usual.
+    /// * `false`: the user will not enroll in new rollouts, and opt out of all existing ones.
+    [Throws=NimbusError]
+    boolean get_rollout_participation();
+
+    [Throws=NimbusError]
+    sequence<EnrollmentChangeEvent> set_rollout_participation(boolean opt_in);
```

Comment:
Let's not make this a breaking change for simplicity.

Let's re-add `set_global_user_participation` and `get_global_user_participation` and mark them deprecated. The implementation for `set` can be to call both `set_experiment_participation` and `set_rollout_participation` with the same value and `get` can just return the value of experiment participation for simplicity.

Then we can proceed to update firefox for ios and firefox for android to use the new APIs and then we can remove the old APIs.

Reply:
Sounds good! I also changed the version back from `0.11.0` to `0.10.1`.